rx/concurrency/mainloopscheduler/eventletscheduler.py

```diff
@@ -72,8 +72,8 @@ def schedule_relative(self,
             (best effort).
         """
 
-        seconds = self.to_seconds(duetime)
-        if not seconds:
+        seconds = max(0.0, self.to_seconds(duetime))
```

Comment:
Yes, I was considering using normalize here as well but figured since we already know it to be float at this point the extra function call with type checking would be spurious compared to a simple max call. Actually even that isn’t necessary here, I realize now, we could simply do `<= 0.0`.

But if you prefer I could use normalize?

About classmethods/staticmethods and whether to invoke them via class or instance, I’m not sure. I follow what I see used at the moment, but might be room for improvement. Perhaps a separate module for temporal utils? SchedulerBase isn’t exported from the concurrency `__init__.py` so it would have to be imported speficially in some places.

I propose addressing that separately though?

Reply:
Ok, we can address this separately. Perhaps just remove `normalize` altogether since it tries to do too much and it's better to use `max` directly (and easier to understand what is going on).

Reply:
Thanks for review! I will merge this now and try to remove `normalize` in another PR.